optimum/exporters/tflite/convert.py

```diff
@@ -173,6 +244,118 @@ def export(
     with TemporaryDirectory() as tmp_dir_name:
         model.save(tmp_dir_name, signatures=signatures)
         converter = tf.lite.TFLiteConverter.from_saved_model(tmp_dir_name)
+
+        if quantization is not None:
```

Comment:
For readability, the quantization case should IMO belong to an other function, but it is taste